contrib/images/slinky.sidecar.dev.Dockerfile

```diff
@@ -17,4 +17,4 @@ COPY --from=builder /src/slinky/build/* /usr/local/bin/
 RUN apt-get update && apt-get install ca-certificates -y
 
 WORKDIR /usr/local/bin/
-ENTRYPOINT ["slinky", "--oracle-config-path", "/oracle/oracle.json", "--market-config-path", "/oracle/market.json"]
+ENTRYPOINT ["slinky", "--oracle-config-path", "/oracle/oracle.json"]
```

Comment:
Do you think it would be helpful to add the tag for update-market-config-path to write the market map when we get one?

Reply:
can probably write to the same directory